opm/autodiff/MissingFeatures.cpp

```diff
@@ -22,12 +22,30 @@
 #include <opm/autodiff/MissingFeatures.hpp>
 #include <unordered_set>
 #include <string>
+#include <map>
 
 
 namespace Opm {
 
 namespace MissingFeatures {
 
+    template<typename T>
+    struct PartiallySupported {
+        int item;
```

Comment:
For increased readability I would prefer if you used item *names* instead - i.e `std::string item`.